packages/oid4vci-demo-frontend/src/configs/kvk.json

```diff
@@ -164,14 +148,6 @@
         "width": 383,
         "height": 383
       },
-      "mobile": {
```

Comment:
Sure SSIInformationRequestPage does not need a mobile scetion?

Reply:
design has no logo here

Reply:
actually 
![Uploading image.png…]()

Reply:
ye i got confussed, it has been moved in the config it seems, under the form